app/lib/cds_importer/entity_mapper/additional_code_description_mapper.rb

```diff
@@ -7,20 +7,16 @@ class AdditionalCodeDescriptionMapper < BaseMapper
 
       self.mapping_path = 'additionalCodeDescriptionPeriod.additionalCodeDescription'.freeze
 
-      self.exclude_mapping = ['validityStartDate', 'validityEndDate'].freeze
+      self.exclude_mapping = %w[validityStartDate validityEndDate].freeze
 
       self.entity_mapping = base_mapping.merge(
         'additionalCodeDescriptionPeriod.sid' => :additional_code_description_period_sid,
         "#{mapping_path}.language.languageId" => :language_id,
         'sid' => :additional_code_sid,
         'additionalCodeType.additionalCodeTypeId' => :additional_code_type_id,
         'additionalCodeCode' => :additional_code,
-        "#{mapping_path}.description" => :description
+        "#{mapping_path}.description" => :description,
       ).freeze
-
-      self.entity_mapping_key_as_array = mapping_with_key_as_array.freeze
```

Comment:
I _think_ this change will mean the mapper keeps recomputing the entity keys. If I was to guess some of this is working around the original developer not understanding class level instance variables

If this is getting removed then `mapping_with_key_as_array` should be memoized at class level, likewise `mapping_keys_to_parse` ?

Reply:
I can add memoization for sure. I'd be blown away if it improved performance at all, though